base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -177,12 +177,14 @@ def inheritance_handler_attributes_domain_add(self, source, specs):
                     new_value = str(expression.AND([old_domain, new_domain]))
                 new_value = self.str2var_domain_text(new_value)
             else:
-                new_value = attribute_node.text
+                # We must ensure that the domain definition has not line breaks because
+                # in update mode the domain cause an invalid syntax error
+                new_value = attribute_node.text.strip()
```

Comment:
```suggestion
                new_value = attribute_node.text.strip().replace("\n")
```

Reply:
@chienandalu we don't need erase blank lines inside text..